### PR TITLE
completed node-fs-writefile exercise.

### PR DESCRIPTION
**Pull request by joseph-w-valdez**
https://user-images.githubusercontent.com/117682160/214718629-b54ed6a9-5a91-4531-8beb-b027bd0b44f0.mp4

### CODE REVIEW

File: node-fs-writefile/note-to-self.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+const args = process.argv;
+let inputs = '';
+for (let index = 2; index < args.length; index++) {
```
Comment (RobertGardner) [inline node-fs-writefile/note-to-self.js:4]: You've cleverly made it work, but it won't work if you have special characters in the note that the terminal wants to interpret. (For example 'this > that' won't work correctly.)

With command line programs, you typically send text parameters by wrapping them in quotes. They will then show up in argv as a single argument.

For example, with:
`node note-to-self "This is my note"`
the note is just the 3rd element in argv.

File: node-fs-writefile/write-rand.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const fs = require('fs');
+const randomNumber = Math.random().toString() + '\n';
```
Comment (RobertGardner) [inline node-fs-writefile/write-rand.js:2]: FYI, when you use '+' if either side is a string then it converts the other side to a string and concatenates. So the `.toString()` call here is redundant.

**Review by RobertGardner — CHANGES_REQUESTED** [review]
Well done, but I think you should make the change I suggested before I Approve.

**Comment by joseph-w-valdez** [comment]
https://user-images.githubusercontent.com/117682160/214909169-3cd4c7b1-6248-4681-9c52-881361dc9720.mp4


https://user-images.githubusercontent.com/117682160/214909186-e9d103e1-966d-495e-8b29-d3aa7277dd45.mp4

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**